### DISCUSSION TITLE
Miner/Python closes after a few days

### DISCUSSION THREAD

**Discussion by gitgryllus** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#298] [Q&A]
Had this problem with v1 miner and now with v2. My laptop that runs the script is always on. But the miner/python closes after a few days; I noticed it first when checking the task manager (the script runs in the background as soon as windows starts). This happens again and again. I have the latest python version. Any guesses? Thx

**Comment by Rakambda** [comment]
Just check the logs. If you see a `CTRL+C` detected then something sends a SIGTERM/SIGKILL to the process.

**Comment by gitgryllus** [comment]
Hello, I am not sure if I view the right logs. I go into the logs folder and there is a log file that has my twitch user's name in the title. But the dates in the log only reach until September. The issue appeared again yesterday. Thx

**Reply by Rakambda** [reply]
Depends how you configured the logger.

**Comment by gitgryllus** [comment]
Can you give me instructions on how to set up the logger so I can get the right info? It is on default settings rn. Thx

**Reply by Rakambda** [reply]
If you use the default settings (https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2#how-to-use) you should have the debug logs in a file.

That file is in a logs folder relative to the working directory.

**Reply by gitgryllus** [reply]
So the log file is in the folder \Twitch-Channel-Points-Miner-v2\logs\username.log
But the entries are all only from 29th of September which makes no sense to me. I will delete the logfile. Hoping to get new logs. Thx

**Reply by gitgryllus** [reply]
p.s. the logging also was set to logging.info so i changed it to logging.debug

**Reply by gitgryllus** [reply]
ok I realized that no logs are created when the script ist started through windows logon automatically, so now I will start it manually and wait until it crashes/closes in the next days.

**Reply by gitgryllus** [reply]
just4info; it's still running. maybe the problem was caused by running the script automatically at windows start. still observing if it will close/crash.

**Reply by gitgryllus** [reply]
It doesn't crash when the script gets started manually. So I will just stick to that. Topic can be closed. BR
